### PR TITLE
Update building stock data of EU datasets

### PR DESCRIPTION
**Pull request by kndehaan**
This PR updates EU datasets as a result of a migration on ETLocal with:

- building stock data, based on EU BSO source
- insulation values for semi-detached houses
- share in useful heat demand per residence type and construction period

Goes with:
- https://github.com/quintel/etsource/pull/3139
- https://github.com/quintel/etlocal/pull/560
- https://github.com/quintel/etdataset/pull/1023
- https://github.com/quintel/etengine/pull/1469

### CODE REVIEW

File: datasets/EU27_european_union_27_countries/EU27_european_union_27_countries.derived.ad [hunk 1]
```diff
@@ -171,28 +171,28 @@
...
-- present_number_of_semi_detached_houses_1965_1984 = 0.0
-- present_number_of_semi_detached_houses_1985_2004 = 0.0
-- present_number_of_semi_detached_houses_2005_present = 0.0
-- present_number_of_terraced_houses_before_1945 = 12958494.44
-- present_number_of_terraced_houses_1945_1964 = 7540000.0
-- present_number_of_terraced_houses_1965_1984 = 26200000.0
-- present_number_of_terraced_houses_1985_2004 = 24683735.1
-- present_number_of_terraced_houses_2005_present = 18090091.9
+- present_number_of_residences = 208874408.0
+- present_number_of_buildings = 10111171.0
```
Comment (kaskranenburgQ) [inline datasets/EU27_european_union_27_countries/EU27_european_union_27_countries.derived.ad:175]: This value should not change. 
When I run the etlocal migration, I see that the same value change happens. While the value has been removed from the migration.
I will try to find out why this keeps happening.

Comment (kaskranenburgQ) [reply datasets/EU27_european_union_27_countries/EU27_european_union_27_countries.derived.ad]: It's fixed now

**Review by kaskranenburgQ — APPROVED** [review]
LGTM!

**Review by kaskranenburgQ — APPROVED** [review]
LGTM!

File: gqueries/general/useful_demand/households/semi_detached_houses/useful_demand_semi_detached_houses_1945_1964.gql [hunk 2]
```diff
@@ -1,3 +1,3 @@
-- query = DIVIDE(V(households_useful_demand_for_space_heating_terraced_houses_1945_1964,demand),BILLIONS)
+- query = DIVIDE(V(households_useful_demand_for_space_heating_semi_detached_houses_1945_1964,demand),BILLIONS)
```
Comment (mabijkerk) [inline gqueries/general/useful_demand/households/semi_detached_houses/useful_demand_semi_detached_houses_1945_1964.gql:1]: Good catch!

**Pull request merged**